x-pack/elastic-agent/pkg/agent/cmd/run.go

```diff
@@ -82,6 +83,10 @@ func run(flags *globalFlags, streams *cli.IOStreams) error { // Windows: Mark se
 		return err
 	}
 
+	if allowEmptyPgp, _ := release.PGP(); allowEmptyPgp {
+		logger.Warn("Package security is disabled. Elastic Agent will not verify signatures of used artifacts.")
```

Comment:
Maybe note that this mean the "Artifact has been build with security disabled"? Because removing that warning is only doable by recompiling.

Reply:
Good point, thanks @ph